Resources/config/provider-resource.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <parameters>
+        <parameter key="cmf_routing.resource.provider.class">Symfony\Cmf\Bundle\RoutingBundle\Resource\ResourceProvider</parameter>
+    </parameters>
+
+    <services>
+
+        <service id="cmf_routing.route_provider" class="%cmf_routing.resource.provider.class%">
```

Comment:
We should probably switch to using aliases rather than each provider taking the main `cmf_routing.route_provider` ID.